kafka-handler/src/test/org/apache/hadoop/hive/kafka/KafkaDagCredentialSupplierTest.java

```diff
@@ -27,17 +27,35 @@
 import org.apache.hadoop.io.Text;
 import org.apache.hadoop.minikdc.MiniKdc;
 import org.apache.hadoop.security.token.Token;
+import org.apache.kafka.common.config.SslConfigs;
+import org.apache.kafka.common.security.auth.SecurityProtocol;
 
+import com.google.common.collect.ImmutableMap;
+
```

Comment:
nit: Are these out of order?

Reply:
Fixed in https://github.com/apache/hive/pull/4430/commits/c3b74d7254d28d8c75f811bbdbb94f11b40e4e6b.

Interestingly, we have no policy, checks, whatsoever on what should be the order of imports in Hive so we end-up with a mixed style with a bit of everything :)

It appears that I was using the import style from another Apache project; now I simply switched to IntelliJ's default cause it matched better the current style in this class.